cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -75,12 +78,18 @@ def from_service_account_file(cls, filename, *args, **kwargs):
     from_service_account_json = from_service_account_file
 
     def __init__(self,
-                 transport=None,
-                 channel=None,
-                 credentials=None,
-                 client_config=None,
-                 client_info=None,
-                 client_options=None):
+                 transport: Union[
```

Comment:
For a type this big would recommend creating a custom type.  QUANTIM_ENGINE_SERVICE_GRPC_TRANSPORT_LIKE?  Oh that feels like Java. Does that fit on one line?